set-version.sh

```diff
@@ -15,3 +15,4 @@ function replace_value_in_file() {
 replace_value_in_file ".*Version = " "$1" "version/version.go" "\""
 replace_value_in_file ".*DefaultKeycloakImage.*= " "quay.io\/keycloak\/keycloak:$1-legacy" "pkg/model/image_manager.go" "\""
 replace_value_in_file ".*image: " "quay.io\/keycloak\/keycloak-operator:$1" "deploy/operator.yaml"
+replace_value_in_file ".*DefaultKeycloakInitContainer.*= " "quay.io\/keycloak\/keycloak-init-container:$1" "pkg/model/image_manager.go" "\""
```

Comment:
I need to double check with @stianst whether the init container will have `-legacy` suffix in the next Keycloak release.